doc/conf.py

```diff
@@ -79,8 +79,9 @@
 # In an ideal world, this would get fixed in this PR:
 # https://github.com/readthedocs/sphinx_rtd_theme/pull/747/files
 def setup(app):
-    app.add_javascript('js/copybutton.js')
-    app.add_stylesheet("basic.css")
+   app.add_js_file('js/copybutton.js')
```

Comment:
Either use single or double quotes consistently, but not both (unless it helps to avoid escaping nested quotes)

Reply:
Fixed